task-json/q3.js

```diff
@@ -0,0 +1,48 @@
+function generateArrayOfNumbers(arrayLength){
+    const arr=new Array(arrayLength);
+    
+
+arr.fill().map((value,indx)=>(arr[indx]=Math.floor(Math.random()*10)));
+
+    return arr ;
+}
+const arr=generateArrayOfNumbers(100);
+function getArrayOfNumbersWithoutDuplication(arrayOfNumbers) {
+    unique = new Set(arrayOfNumbers);
+    console.log(unique);
+    return unique
+}
+
+function getArrayOfNumbersWithoutDuplication1(arrayOfNumbers) {
+
+uniqueArray = arrayOfNumbers.filter(function(item, pos) {
+    return arrayOfNumbers.indexOf(item) == pos;
+})
+
+console.log(uniqueArray);
+
+return uniqueArray;
+}
+function getDuplication(arrayOfNumbers) {
+    const dublicte= new Map(); 

```

Comment:
this function will return array of duplicated numbers, not their count

Reply:
it will return a map with key and value 
key is the number and value is their count